tests/ci/codebuild/release/linux_aarch64_build.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+    commands:
+      - yum -y update
+      - yum -y group install 'Development Tools'
+      - curl -L -O https://downloads.sourceforge.net/ltp/lcov-1.14-1.noarch.rpm
+      - yum -y install cmake3; alias cmake=cmake3
```

Comment:
do you need to alias this if you're specifying the cmakeBin override property?

Reply:
Forgot to remove it here. Shall do.